core/src/main/resources/xml/xsl/uglify_lookup.xml

```diff
@@ -49,11 +49,6 @@
     <Type>listener</Type>
     <ClassName>nl.nn.adapterframework.http.HttpListener</ClassName>
   </Element>
-  <Element>
-    <Name>IfsaProviderListener</Name>
-    <Type>listener</Type>
-    <ClassName>nl.nn.adapterframework.extensions.ifsa.ejb.IfsaProviderListener</ClassName>
-  </Element>
```

Comment:
Hier moet niet de IfsaProviderListener er helemaal uit vallen, maar vervangen worden door nl.nn.adapterframework.extensions.ifsa.jms.IfsaProviderListener.

Reply:
uglify_lookup.xml is generated, like ibisdoc.xsd is. In an earlier version of this pull request, I could get IfsaProviderListener included, but I did so by unwanted changes to the pom.xml files. Gerrit, we had a phone call about this. You said you agree now that IfsaProviderListener is omitted.